WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionWebViewActivity.java

```diff
@@ -20,12 +22,17 @@
 
     public enum Source {
         STATS("stats"), NOTIFICATIONS("notifications");
+        public static final String KEY = "tracking_source";
```

Comment:
This is a bit of a nitpick, but do you think it'd be better to have this `KEY` in the activity instead? Aside from `WordPress.Site` which is used everywhere, all our other keys are in the activity/fragments as far as I know and it might be better to follow that pattern here for consistency. However, if you feel strongly about having it here, I am OK with that.

Reply:
I'm fine with moving it. Somehow it feels a bit more natural to me to have a key like Source.KEY than Activity.SOURCE_KEY but I'm up for that for consistency.